src/components/core/Alert.vue

```diff
@@ -19,12 +19,15 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
   <div v-if="alert">
     <v-alert
       :value="true"
-      :color="alert.getColor()"
+      :type="alert.getColor()"
       :icon="alert.getIcon()"
-      class="ma-0"
+      :class="getColor(alert.getColor())"
       dismissible
+      tile
+      light
+      colored-border
```

Comment:
`tile` removes the rounded borders.
`light` and `colored-border` to indicate no dark mode (due to upstream bug)